kotlin/hello-world-lambda-cdk/lambda/src/main/resources/log4j2.properties

```diff
@@ -0,0 +1,12 @@
+# Set to debug or trace if log4j initialization is failing
```

Comment:
Perhaps can you check whether we can configure this? https://github.com/restatedev/documentation/issues/184

Reply:
On it!